tests/unit_test/chat/chat_test.py

```
@@ -2088,41 +2088,57 @@ def test_delete_media_from_bsp_channel_does_not_exist():
 
     assert str(exc_info.value) == "Media deletion failed: No channel found for this bot. Please configure the channel first."
 
-from datetime import datetime, timedelta
-from croniter import croniter
-
-# Import your function
+from datetime import datetime
```

Ruff (0.14.10)

Redefinition of unused `datetime` from line 2

Remove definition: `datetime`

(F811)